### PR TITLE
Sanctions: Weapons for PS Legacy update

### PR DESCRIPTION
**Pull request by fredkilbourn**
* Add new Archer variants
* Add new Apex variants
* Move all archers into new anti-material category
* Reclassify underwater category into new amphibious categories
* Add sanctions to new weapons

Signed-off-by: Fred Kilbourn <fred@fredk.com>

### CODE REVIEW

**Comment by brakenium** [comment]
Is there any reason for AMR's to be classified as infantry when sniper rifles are sniper? I'd say it should either be sniper or an AV sanction

**Comment by fredkilbourn** [comment]
> Is there any reason for AMR's to be classified as infantry when sniper rifles are sniper? I'd say it should either be sniper or an AV sanction

This is how it was back in the Das Anfall days, we've just followed how it used to be and nobody has questioned it yet.

**Comment by fredkilbourn** [comment]
I feel if we can get some interested parties to discuss it we can consider changing them, but for now that's what they are and have been since the beginning.

**Comment by RemainNA** [comment]
I think given the headshot changes it makes sense to reclassify them at this point

**Comment by RemainNA** [comment]
Between the flack of the Masthead and all of them being intended for AV I'd go with `none`

**Comment by brakenium** [comment]
I'd personally go with `sniper` since in my opinion and from what I hear from others is that they find the archer less frustrating to deal with because you're not dealing with a cloaker. I also don't think that it's armour damage is as powerful as say a rocket launcher. However the new AMR's might fit the `none` category better since they seem to have some questionable faction traits, especially the NC variant.

TLDR; Archer and variants might be better as `sniper` while we should wait with the new faction specific AMR's and set them to `none` until we have a better understanding of how they play out on live

**Comment by fredkilbourn** [comment]
I'm leaning toward `none` personally, I feel like they taint the `sniper` classification by being too different than the other generic bolt action snipers.  They will sway vehicle kdr numbers, shot to kill numbers from the long time they were a much lower damage model, etc.  I'll wait to hear more discussion but I'm leaning harder this way now.

**Comment by brakenium** [comment]
How about giving them their own sanction? They do seem to be more prevalent now

**Comment by fredkilbourn** [comment]
Creating outright new sanctions will take a larger consensus of the dev community, last time we did that was for `max` and `meme` I think...

**Comment by fredkilbourn** [comment]
Since we're not getting a lot of discussion on this I'm going to merge this as-is with the sanctions set based on historical equivalents.  If anyone wants to pursue a discussion on the matter please open an issue and try to gather interest from relevant people.
